test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -114,29 +124,42 @@ public async Task GivenPatientsWithSameBirthdateAndMultiplePages_WhenSortedByBir
             // For SQL, we always choose the "oldest" resource based on last updated time (irrespective of overall sort order)
             // Since sort is based on same BirthDate, the order in which resources will be returned depends on their creation time for both sortParameterName
             // Above patients array could be out of sync due to inconsistent system time across multiple server instances so sort the expected patients array
-            await ExecuteAndValidateBundle($"Patient?_tag={tag}&_sort={sortParameterName}&_count=3", false, pageSize: 3, patients.OrderBy(p => p.Meta.LastUpdated).ToArray());
+            var returnedResults = await GetResultsFromAllPagesAsync($"Patient?_tag={tag}&_sort={sortParameterName}&_count=3");
+
+            if (sortParameterName == "birthdate")
+            {
+                SortTestsAssert.AssertPatientBirthDateAscendingOrderInRange(0, 3, returnedResults.Take(3).ToList());
```

Comment:
This is testing that the sort works over multiple pages, but only the first page of results is being checked. Same for the two Cosmos tests below.

Reply:
Yeap. I've noticed that. I was trying to keep the original test assert, as it was doing that. I've changed the code to compare everything.